pyccel/stdlib/math/pyc_math.f90

```diff
@@ -0,0 +1,19 @@
+module math
+    implicit none
+
+        real, parameter,private :: pi = 3.1415926536
+        real, parameter, private :: e = 2.7182818285
```

Comment:
I think we should save these in double precision. Most of the generated code will use double precision